shared/js/content-scripts/block-cookie.js

```diff
@@ -123,12 +110,18 @@
      * @param {Function} func Function to run
      * @param {string} arg Optional argument to pass to the function
      */
-    function inject (func, arg) {
-        const scriptString = `(${func.toString()})('${arg}')`
+    function inject (func, arg, requires = []) {
+        const imports = requires.map(func => func.toString()).join(';\n')
+        const scriptString = `{
+            ${imports};
+            (${func.toString()})('${arg}')
+        }`
         const doc = window.wrappedJSObject ? window.wrappedJSObject.document : document
         const scriptElement = doc.createElement('script')
         scriptElement.innerHTML = scriptString
         doc.documentElement.prepend(scriptElement)
+        // remove element immediately so it is not visible to scripts on the page
+        doc.documentElement.removeChild(scriptElement)
```

Comment:
oh, we can do it sync? I thought this would require `setTimeout(,0)` or `script.onload`?

Reply:
I also thought the same, but when I tested it sync also worked.